packages/eligibility/src/Eligibility.js

```diff
@@ -19,6 +19,20 @@ function Eligibility() {
   const [wrongQuestions, setWrongQuestions] = useState([]);
   const [resultClick, setResultClick] = useState(null);
 
+  React.useEffect(() => {
+    document.addEventListener('keydown', handleKeys);
+
+    return () => {
+      document.removeEventListener('keydown', handleKeys);
+    };
+  }, []);
+
+  function handleKeys(e){ 
+    console.log("Enter handle keys : " + e.keyCode);
```

Comment:
If this is production-ready code, kindly remove `console.log()` statements.

Reply:
Resolved in commit 8ca0f51.

Thank you @lacabra!